basemaps/layers.md

```diff
@@ -11,100 +11,144 @@ outline: deep
 OpenStreetMap layers documentation.
 
 ::: warning
-This section is under construction.
-
-For the previous version of the layers, see [Layers Version 2](/basemaps/layers_v2)
+This section is under construction for Version 4.
 :::
 
-<MaplibreMap/>
-
 ## boundaries
 
-| Key                    | Values                                    | Description |
-| ---------------------- | ----------------------------------------- | ----------- |
-| `pmap:kind`            | `country`, `region`, `county`, `locality` |             |
-| `pmap:kind_detail`     |                                           |             |
-| `pmap:min_admin_level` |                                           |             |
-| `pmap:brk_a3`          |                                           |             |
-| `disputed`             | boolean                                   |             |
+<MaplibreMap highlightLayer="boundaries"/>
+
+| Key               | Values                                    | Description |
+| ----------------- | ----------------------------------------- | ----------- |
+| `kind`            | `country`, `region`, `county`, `locality` |             |
+| `kind_detail`     |                                           |             |
```

Comment:
It's possible `kind_detail` should have the contents of `min_admin_level`, and the 2nd removed in v4?

Reply:
(This could be why the Tilezen styles aren't rendering boundary lines at low zooms...)

Reply:
Will do. What about `brk_a3`?